### DISCUSSION TITLE
Dati padre su casi d'uso Dic_Nasc_063, Dic_Nasc_066, Dic_Nasc_072

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#889] [Supporto servizi cooperativi]
Nei casi d'uso:

![image](https://github.com/italia/ansc/assets/26415430/f2e36466-ae19-4eac-a2d7-dde236a56d35)

non è attualmente possibile trasmettere in nessun caso i dati del padre.
Non ci risulta però che, anche laddove il riconoscimento antecedente alla nascita venga eseguito solo dalla madre, all'atto della denuncia di nascita non possa essere presente (e riconoscere il nato) anche il padre.
Se , diversamente, i casi d'uso in oggetto fanno volutamente riferimento al caso in cui il padre non sia presente neanche al momento della nascita, a quali casi d'uso dovremmo ricondurre gli atti dello stesso tipo ma con la presenza (ed il riconoscimento alla nascita da parte) del padre?

Questione del tutto analoga si pone per i casi d'uso:
Dic_Nasc_109
Dic_Nasc_075

**Comment by gbuemi** [comment]
Buongiorno @antonioluberto,
al momento non è previsto trasmettere il padre e non sono previsti casi d'uso analoghi con la presenza del padre. Faremo un passaggio con Anusca per approfondire la questione.

Grazie e saluti,
Supporto ANSC
